libraries/AC_PID/AC_PID_Basic.cpp

```diff
@@ -0,0 +1,176 @@
+/// @file	AC_PID_Basic.cpp
+/// @brief	Generic PID algorithm
+
+#include <AP_Math/AP_Math.h>
+#include "AC_PID_Basic.h"
+
+#define AC_PID_Basic_FILT_E_HZ_DEFAULT 20.0f   // default input filter frequency
+#define AC_PID_Basic_FILT_E_HZ_MIN     0.01f   // minimum input filter frequency
+#define AC_PID_Basic_FILT_D_HZ_DEFAULT 10.0f   // default input filter frequency
+#define AC_PID_Basic_FILT_D_HZ_MIN     0.005f  // minimum input filter frequency
+
+const AP_Param::GroupInfo AC_PID_Basic::var_info[] = {
+    // @Param: P
+    // @DisplayName: PID Proportional Gain
+    // @Description: P Gain which produces an output value that is proportional to the current error value
+    AP_GROUPINFO("P",    0, AC_PID_Basic, _kp, 0),
+
+    // @Param: I
+    // @DisplayName: PID Integral Gain
+    // @Description: I Gain which produces an output that is proportional to both the magnitude and the duration of the error
+    AP_GROUPINFO("I",    1, AC_PID_Basic, _ki, 0),
+
+    // @Param: IMAX
+    // @DisplayName: PID Integral Maximum
+    // @Description: The maximum/minimum value that the I term can output
```

Comment:
is it misleading?  I think the imax is a limit on the I term's contribution to the output.  So I think it's in the units of the output.. 

I think I might leave this one as-is because it is also consistent with the comment on all the other PID classes.